client/.eslintrc.cjs

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  ignorePatterns: ["dist"],
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module',
+    project: 'client/tsconfig.json'
```

Comment:
Not sure this is taking the config file properly

Reply:
🤔 this is correct to run eslint from the root dir. Like in `eslint .`

How are you trying to run eslint? From the `client` dir in this case?

Reply:
Yup, as expected

<img width="524" alt="Screenshot 2022-05-02 at 12 19 55" src="https://user-images.githubusercontent.com/2181866/166219693-77aec28c-7922-4a36-95fe-6b7e6d509bbe.png">